zebra-state/src/service/finalized_state/zebra_db/shielded.rs

```diff
@@ -402,19 +399,32 @@ impl DiskWriteBatch {
             self.zs_insert(&orchard_tree_cf, height, trees.orchard);
         }
 
-        // TODO: Increment DATABASE_FORMAT_MINOR_VERSION and uncomment these insertions
-
-        // if let Some(subtree) = trees.sapling_subtree {
-        //     self.zs_insert(&sapling_subtree_cf, subtree.index, subtree.into_data());
-        // }
+        if let Some(subtree) = trees.sapling_subtree {
+            self.insert_sapling_subtree(zebra_db, subtree);
```

Comment:
Note: Since this PR writes a new format to the database, it can't merge until after the 1.2.0 release.